ArduCopter/Parameters.cpp

```diff
@@ -1215,6 +1215,16 @@ const AP_Param::GroupInfo ParametersG2::var_info2[] = {
     // @User: Advanced
     AP_GROUPINFO("TKOFF_THR_MAX", 6, ParametersG2, takeoff_throttle_max, 0.9),
 
+#if MODE_FLIP_ENABLED == ENABLED
+    // @Param: FLIP_RATE
+    // @DisplayName: Rotational Rate for Flip Mode
```

Comment:
Not a blocker but putting "Flip mode" at the beginning of the description would be good.  So maybe, "Flip Mode Rotational Rate"

Reply:
Done.  Thanks!